sync-core/src/main/java/com/cloudant/android/Base64OutputStreamFactory.java

```diff
@@ -28,7 +28,7 @@ public static OutputStream get(OutputStream os) {
             if (Misc.isRunningOnAndroid()) {
                 Class c = Class.forName("android.util.Base64OutputStream");
                 Constructor ctor = c.getDeclaredConstructor(OutputStream.class, int.class);
-                return (OutputStream)ctor.newInstance(os, 0);
+                return (OutputStream)ctor.newInstance(os, 2);
```

Comment:
Can we statically use http://docs.oracle.com/javase/7/docs/api/java/lang/Class.html#getDeclaredField%28java.lang.String%29 to get the field value for NO_WRAP?